fastlane_core/lib/fastlane_core/itunes_transporter.rb

```diff
@@ -568,9 +690,11 @@ def upload(app_id = nil, dir = nil, package_path: nil, asset_path: nil)
 
       password_placeholder = @jwt.nil? ? 'YourPassword' : nil
       jwt_placeholder = @jwt.nil? ? nil : 'YourJWT'
+      api_key_plaseholder = nil unless @api_key.nil?
```

Comment:
```suggestion
      api_key_placeholder = nil unless @api_key.nil?
```

please fix this spell in other places too...